### PR TITLE
REQUEST FOR CODE REVIEW OF REACT MATH MAGICIANS UNIT AND INTEGRATION TESTING PROJECT

### PR DESCRIPTION
**Pull request by Moreneecoder**
In this code, I wrote unit tests for the `calculate.js` module to ensure all functionalities for calculation works as expected.
I also wrote integration tests for the following components using jest snapshots:
* Home.js
* App.js
* Quote.js

### CODE REVIEW

File: src/__test__/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// import { render, screen } from "@testing-library/react"
```
Comment (rukundoeric) [inline src/__test__/Home.test.js:1]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/__test__/Home.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+// test(`on render, page should contain h1 of 'Where Magic Of Maths Is Stripped Bare'`, () => {
+//     render(<Home />)
+
+//     // screen.debug();
+//     expect(screen.getByRole('heading',
+//     {name: /Where The Magic Of Math Is Stripped Bare/}))
+//     .toBeInTheDocument()
+
+// })
```
Comment (rukundoeric) [inline src/__test__/Home.test.js:13]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

**Review by rukundoeric — CHANGES_REQUESTED** [review]

File: src/__test__/Home.test.js (hunk 1 above)
Comment (rukundoeric) [inline src/__test__/Home.test.js:1]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/__test__/Home.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (rukundoeric) [inline src/__test__/Home.test.js:14]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Moreneecoder, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by e71az — APPROVED** [review]
## Status:  Approved 🟢

Hi, @Moreneecoder, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**